<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Clerk</class>
 <widget class="QMainWindow" name="Clerk">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1505</width>
    <height>919</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="ClerkMain">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>0</y>
      <width>1511</width>
      <height>891</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="CLerkImg">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>30</y>
        <width>671</width>
        <height>861</height>
       </rect>
      </property>
      <property name="text">
       <string>IMG</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>270</y>
        <width>171</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Repair Requests</string>
      </property>
      <widget class="QPushButton" name="Create_RepairRequests_Button">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>83</width>
         <height>29</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Update_RepairRequests_Button">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>83</width>
         <height>29</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Logged in as Clerk User</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>430</y>
        <width>171</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Reports</string>
      </property>
      <widget class="QPushButton" name="Reports_Section_Button">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>83</width>
         <height>29</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Reports</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Go_Reports_Button">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>83</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Go</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>300</y>
        <width>181</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Maintenance Request</string>
      </property>
      <widget class="QPushButton" name="Create_MaintenanceRequests_Button">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>83</width>
         <height>29</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Update_MaintenanceRequests_Button">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>83</width>
         <height>29</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>60</y>
        <width>181</width>
        <height>221</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Rental Requests</string>
      </property>
      <widget class="QPushButton" name="Create_RentalRequests_Button">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>83</width>
         <height>29</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Update_RentalRequests_Button">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>83</width>
         <height>29</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Delete_RentalRequests_Button">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>83</width>
         <height>29</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Approve_RentalRequests_Button">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>83</width>
         <height>29</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Approve</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>70</y>
        <width>171</width>
        <height>161</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Customers</string>
      </property>
      <widget class="QPushButton" name="Create_Customers_Button">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>83</width>
         <height>29</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Update_Customers_Button">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>83</width>
         <height>29</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Delete_Customers_Button">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>83</width>
         <height>29</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>0</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Create Customer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="CreateCustomer_Back">
      <property name="geometry">
       <rect>
        <x>1400</x>
        <y>0</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QPushButton" name="UpdateCustomer_Back">
      <property name="geometry">
       <rect>
        <x>1400</x>
        <y>0</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>0</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Update Customer</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>0</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Delete Customer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="DeleteCustomer_Back">
      <property name="geometry">
       <rect>
        <x>1400</x>
        <y>0</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>0</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Create Repair Requests</string>
      </property>
     </widget>
     <widget class="QPushButton" name="CreateRepair_Back">
      <property name="geometry">
       <rect>
        <x>1400</x>
        <y>0</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_6">
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>0</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Update  Repair Requests</string>
      </property>
     </widget>
     <widget class="QPushButton" name="UpdateRepair_Back">
      <property name="geometry">
       <rect>
        <x>1400</x>
        <y>0</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_7">
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>0</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Create Rental Requests</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Create_Rental_Request_Back">
      <property name="geometry">
       <rect>
        <x>1400</x>
        <y>0</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_8">
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string> Update Rental Requests</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Update_Rental_Request_Back">
      <property name="geometry">
       <rect>
        <x>1400</x>
        <y>0</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_9">
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string> Delete Rental Requests</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Delete_Rental_Request_Back">
      <property name="geometry">
       <rect>
        <x>1400</x>
        <y>0</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_10">
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string> Approve Rental Requests</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Approve_Rental_Request_Back">
      <property name="geometry">
       <rect>
        <x>1400</x>
        <y>0</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_11">
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>221</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Create Maintenance Requests</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Create_Maintenance_Request_Back">
      <property name="geometry">
       <rect>
        <x>1400</x>
        <y>0</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_12">
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>221</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Update Maintenance Requests</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Create_Maintenance_Request_Back_2">
      <property name="geometry">
       <rect>
        <x>1400</x>
        <y>0</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_13">
     <widget class="QPushButton" name="Reports_Back">
      <property name="geometry">
       <rect>
        <x>1400</x>
        <y>0</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>10</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Reports Section</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>850</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Update  Repair Requests</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>850</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Create Rental Requests</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1505</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
